### ISSUE TITLE
BUG: repeated mirror images  retry run

### ISSUE THREAD

**Issue by cubxxw** [labring/sealos#2066]
### Sealos Version

latest

### How to reproduce the bug?

There is a small bug, but thankfully the mirror is not duplicated？
![image](https://user-images.githubusercontent.com/86140903/200563054-a0ddb195-5532-4aa7-b80c-3531fa666f75.png)

### What do you see instead?

  image:
  - labring/kubernetes-docker:v1.20.5-4.1.3
  - labring/calico:v3.24.1
  - labring/kubernetes:v1.25.0
  - labring/helm:v3.8.2
  - labring/calico:v3.24.1
  - labring/kubernetes:v1.25.0
  - labring/helm:v3.8.2
  - labring/calico:v3.24.1
  - labring/kubernetes:v1.25.0
  - labring/helm:v3.8.2
  - labring/calico:v3.24.1
  - labring/kubernetes:v1.25.0
  - labring/helm:v3.8.2
  - labring/calico:v3.24.1

### Operating environment

```markdown
- Sealos version:
- Docker version:
- Kubernetes version:
- Operating system:
- Runtime environment:
- Cluster size:
- Additional information:
```

**Comment by cuisongliu** [comment]
Do you describe it in more detail, such as how you execute commands?

**Comment by 3293172751** [comment]
Or yep! Sorry, I didn't describe it clearly. The command I used is as follows:

` ` ` bash
Sealer run labring/kubernetes: v1.25.0 labring/helm: v3.8.2 labring/calico: v3.24.1 - in use
` ` `

Don't know what the reason, download is slow, and `labring/kubernetes: v1.25.0` to download for the third time to *success*, of course, the accumulation of a lot of mirror, just as you said see above, the command is:

` ` ` bash
The cat ~ /. Sealos/default/Clusterfile | less
` ` `

![image](https://user-images.githubusercontent.com/86140903/200767037-6b0a3631-6d13-45e8-98ae-bfc9fb075e54.png)

💡 I wonder if these repeated mirror images are superposition, I hope it can be solved ~

**Comment by fanux** [comment]
You can fix this in a separate PR @3293172751

**Comment by 3293172751** [comment]
yep , thanks 🏄‍♂️ 

I think I'd love to do this `pr`, but right now I'm just a rookie, still flying around the `sealos` forest. I hope someone can help me. @fanux

**Comment by fengxsong** [comment]
Will be fixed in #2163
